appliances/empty-vm.gns3a

```diff
@@ -2,13 +2,13 @@
     "appliance_id": "1cfdf900-7c30-4cb7-8f03-3f61d2581633",
     "name": "Empty VM",
     "category": "guest",
-    "description": "An empty VM with empty hard disks 8G, 30G & 100G.",
+    "description": "A empty VM with empty hard disks 8G, 30G, 100G & 200G.",
     "vendor_name": "GNS3",
     "vendor_url": "https://gns3.com",
     "documentation_url": "",
     "product_name": "QEMU",
     "product_url": "https://sourceforge.net/projects/gns-3/files/Empty%20Qemu%20disk/",
-    "registry_version": 4,
+    "registry_version": 5,
```

Comment:
Is there a reason to go for registry_version 5 ?

Reply:
I thought it was a increment to each update